src/main/windows/editor.js

```diff
@@ -47,11 +47,11 @@ class EditorWindow extends BaseWindow {
 
     // Enable native or custom/frameless window and titlebar
     const { titleBarStyle } = preferences.getAll()
-    if (titleBarStyle === 'custom') {
-      winOptions.titleBarStyle = ''
-    } else if (titleBarStyle === 'native') {
-      winOptions.frame = true
-      winOptions.titleBarStyle = ''
+    if (!isOsx) {
```

Comment:
@Jocs You could use `winOptions.titleBarStyle = 'csd'` on macOS and remove `!isOSX` in the renderer.

Reply:
I preferred add `isOsx` in render, because it maybe show the `Windows controls` if user change the `titleBarStyle` to custom in the `preferences.json` file,.

Reply:
Could we add a variable `showCustomTitleBar: boolean`?

Reply:
computed prop in vue?

Reply:
Yes, a vue property.